database/knexfile.js

```diff
@@ -3,6 +3,7 @@ const env = require('./database-env')
 module.exports = {
   client: 'pg',
   connection: env.PGURI,
+  searchPath: [env.PGSCHEMA],
```

Comment:
@k1ch  should we add `public` as a second element to this array?

Reply:
Sure.